inference/nlp/realtime/llm/opt30b/djl_deepspeed_deploy_opt30b_no_custom_inference_code.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<p>The license says it can't be used for commercial problems - SageMaker Example Notebooks is a commercial pursuit, so you can't use this model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>